### DISCUSSION TITLE
Invalid value DATA_KIOSK_QUERY_PROCESSING_FINISHED for path parameter notificationType

### DISCUSSION THREAD

**Discussion by cohlar** [amzn/selling-partner-api-models#3925] [General]
Hi,

I am following the documentation to [subscribe to the DATA_KIOSK_QUERY_PROCESSING_FINISHED notification](https://developer-docs.amazon.com/sp-api/v0/docs/data-kiosk-notification), and I am getting the following error from the `getSubscription` operation: 

> Invalid value DATA_KIOSK_QUERY_PROCESSING_FINISHED for path parameter notificationType

I know the Data Kiosk is in beta, but since the notification is documented, I'm here to ask whether this notification cannot yet be subscribed to (that would explain the error message), or if on the contrary it is functional and I am doing something wrong.

Thanks for your help.

**Comment by cohlar** [comment]
@shreeharsh-a converting this issue into a discussion only makes sense if the DATA_KIOSK_QUERY_PROCESSING_FINISHED notification is up and running - can you please confirm if this is the case?

Otherwise, the facts that the notification is documented, included in the models / schemas, and does not work, make it a dev issue that requires fixing (either make the notification work or update the documentation to reflect the current status).
The fact that DataKiosk is in BETA doesn't justify documenting things that don't work...

**Comment by shreeharsh-a** [comment]
Please reach out to [developer support](https://developer.amazonservices.com/support) so that we can better help you with issues related to SP API.

`Note: SP API or docs related issues / troubleshooting support is managed by a different team. We can help if the issue is related to the content published on this repo.`
